addon/-private/system/store.js

```diff
@@ -233,25 +233,6 @@ Store = Service.extend({
   adapter: '-json-api',
 
   /**
-    Returns a JSON representation of the record using a custom
-    type-specific serializer, if one exists.
-
-    The available options are:
-
-    * `includeId`: `true` if the record's ID should be included in
-      the JSON representation
-
-    @method serialize
-    @private
-    @param {DS.Model} record the record to serialize
-    @param {Object} options an options hash
-  */
-  serialize(record, options) {
```

Comment:
@sly7-7  do you mind deprecating this instead of removing it? Just in case someone was using it.